sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -52,50 +52,36 @@ object ScalaNativePluginInternal {
     taskKey[Unit]("Warn if JVM 7 or older is used.")
 
     lazy val scalaNativeDependencySettings: Seq[Setting[_]] = {
-      val organization = "org.scala-native"
       val nativeStandardLibraries =
         Seq("nativelib", "clib", "posixlib", "windowslib", "javalib", "auxlib")
 
       Seq(
-        libraryDependencies ++= Seq(
-          organization %%% "test-interface" % nativeVersion % Test
-        ),
-        libraryDependencies += CrossVersion
-          .partialVersion(scalaVersion.value)
-          .fold(throw new RuntimeException("Unsupported Scala Version")) {
-            case (2, _) =>
-              organization %%% "scalalib" % scalalibVersion(
-                scalaVersion.value,
-                nativeVersion
-              )
-            case (3, _) =>
-              organization %%% "scala3lib" % scalalibVersion(
-                scalaVersion.value,
-                nativeVersion
-              )
-          },
-        libraryDependencies ++= nativeStandardLibraries.map(
-          organization %%% _ % nativeVersion
-        ),
+        libraryDependencies ++= {
+          val org = nativeOrgName
```

Comment:
Let's decouple a current dependency between our build definition and the sbt plugin we publish. 
Let's simply inline `nativeOrgName` in exactly 2 places - here and in the ThisBuild / version definition. All other usages refer to these two.